css-frontend/src/main/java/org/sonar/css/model/property/standard/ColumnCount.java

```diff
@@ -22,10 +22,13 @@
 import org.sonar.css.model.property.StandardProperty;
 import org.sonar.css.model.property.validator.ValidatorFactory;
 
+import java.lang.System;
+
 public class ColumnCount extends StandardProperty {
 
   public ColumnCount() {
     setExperimental(true);
+    System.out.println("qsfljsqlmfjqs");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)